index.css

```diff
@@ -0,0 +1,208 @@
+    margin: 0 auto;
+    align-items: flex-start;
+    justify-content: center;
+    
+
+}
+
+main {
+    /* border: 2px solid blue; */
+    padding: 0.5em;
+    margin: 0.5em;
+}
+
+.code-preview {
+    /* border:2px solid orange; */
+    padding: 0.5em;
+    margin: 0.5em;
+}
+
+.main-container {
+    width: 675px;
+    margin: 0 auto;
+}
+
+nav>ul{
+    display: flex;
+    flex-wrap: wrap;
+
+   }
+
+nav>ul>li{
+  margin:1em;
+}
+
+nav>ul>li>a{
+    text-decoration: none;    
+    padding: 0.5em;
+    background-color: var(--secondary-color);
+    color:white;
+}
+
+nav>ul>li>a:hover{
+    background-color: white;
+    color:var(--secondary-color);
+    border: 2px solid var(--tertiary-color);
+}
+aside {
+    /* border: 2px solid green; */
+    float: left top;
+    text-align: left;
+    padding: 0.5em;
+    margin: 0.5em;
+    border-right: 2px solid var(--secondary-color);
+  
+}
+
+aside>ul>li {
+    padding: 0.5em;
+    list-style-type: none;
+    margin: 1em;
+}
+
+aside>ul>li>a {
+    text-decoration: none;    
+    padding: 0.5em;
+    background-color: var(--secondary-color);
+    color:white;
+
+}
+
+aside>ul>li>a:hover{
+    background-color: white;
+    color:var(--secondary-color);
+    border: 2px solid var(--tertiary-color);
+}
+footer {
+    text-align: center;
+    /* border: 2px solid magenta; */
+    margin: 0.5em;
+    padding: 0.25em;
+    background-color: var(--secondary-color);
+    color:white;
+}
+
+.current-page {
+    border: 2px solid var(--tertiary-color);
+    border-radius:1em;
+   background-color: var(--current-selected-color);
+   color:var(--secondary-color);
+}
+
+.component-main-display {
+
+    padding: 0.5em;
+    margin: 0.5em;
+
+}
+
+.component-container {
+   display:flex;
+   justify-content: space-between;
+   flex-direction: column;
+   
+}
+
+.component-description{
+    letter-spacing: 1px;
+    
+}
+
+.textColor-btn-container,
+.button-sizes-container,
+.squareEdge-bordered-btn-container,
+.squareEdge-filled-btn-container,
+.roundEdge-btn-container,
+.button-style-container,
+.icon-btn-sizes-container,
+.icon-btn-container,
+.badge-on-icon-container,
+.text-badge-container, 
+.rating-sizes{
+    display: flex;
+    justify-content: space-evenly;
+    flex-wrap:wrap;
+    align-items: flex-start;
+   
+    
+}
+.textColor-btn-container>section,
+.button-sizes-container>section,
+.squareEdge-bordered-btn-container>section,
+.squareEdge-filled-btn-container>section,
+.roundEdge-btn-container>section,
+.button-style-container>section,
+.icon-btn-sizes-container>section,
+.icon-btn-container>section,
+.badge-on-icon-container>div,
+.rating-sizes>section{
+    margin:0.5em 1em 0.5em 1em;
+}
+.about-image-content{
+    display: flex;
+    margin:1em;
+    justify-content: space-between;
+}
+.about-image-content>img{
+    width:450px;
+}
+
+@media (max-width:675px) {
+
+    body, header,main,nav,footer, aside,.main-container,.container, .component-main-display,.component-container{
```

Comment:
Here instead of using px, you may consider using % unit for width. 

If you define it in px it will have a fixed width. However if you define it in % it will be relative to the screen's width (or to the containing element).
 I would prefer to use % as it guarantees that it will (relatively) look the same on every screen

Go through this for more details: https://stackoverflow.com/questions/32307862/what-us-difference-between-width-in-and-width-in-px#:~:text=One%20point%20is%20equal%20to,and%20cannot%20scale%20in%20size.&text=If%20you%20define%20it%20in,or%20to%20the%20containing%20element).

![image](https://user-images.githubusercontent.com/46996877/112790091-2a99d080-907c-11eb-86a7-94e0bbd7d878.png)